tests/components/simplepush/test_config_flow.py

```diff
@@ -0,0 +1,89 @@
+"""Test Simplepush config flow."""
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.simplepush.const import CONF_DEVICE_KEY, CONF_SALT, DOMAIN
+from homeassistant.const import CONF_NAME, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+MOCK_CONFIG = {
+    CONF_DEVICE_KEY: "abc",
+    CONF_NAME: "simplepush",
+    CONF_PASSWORD: "password",
+    CONF_SALT: "salt",
+}
+
+
+async def test_flow_successful(hass: HomeAssistant) -> None:
+    """Test user initialized flow."""
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": config_entries.SOURCE_USER},
+    )
+    result = await hass.config_entries.flow.async_configure(
```

Comment:
We normally patch the integration setup functions on all create entry results to avoid setting up the whole integration. See the scaffold script test template for example.